unity/Assets/Scenes/MockData.unity

```diff
@@ -1607,7 +1607,7 @@ MonoBehaviour:
   m_OnCullStateChanged:
     m_PersistentCalls:
       m_Calls: []
-  m_text: "localhost:8500\u200B\u200B"
+  m_text: "localhost:8500\u200B"
```

Comment:
Hmmm, I wonder what is the \u200B thing?

Reply:
I think this is a hidden character; with this I couldn't press connect (bugged me out for a few days) so deleting it resolved the issue. I think unity auto-added the character to the end of a TMP text input box